lib/messaging/messagehandler_message.rb

```diff
@@ -56,24 +56,20 @@ def move_relative(message)
       0, # time
       0, # external_info
     ]
-    $bot_dbaccess.create_new_command(Time.now,'menu')
-    $bot_dbaccess.add_command_line(*args)
-    $bot_dbaccess.save_new_command
-    $bot_dbaccess.increment_refresh
   rescue => e
     puts e.message, e.backtrace.first
   end
 
   def home_x(*)
-    raise 'Not implemented yet.'
+    HardwareInterface.current.move_home_x
```

Comment:
Actually messagehandler_message.rb should not have any code for handling messages, as the code to to that is already in /messagehandler_schedule.rb. That is, if you use the single_command message type and not the move_rel.